src/journey-maps/angular/journey-maps.interfaces.ts

```diff
@@ -25,6 +25,8 @@ export interface SbbInteractionOptions {
   oneFingerPan?: boolean;
   /** Whether the map can be zoomed by scrolling. */
   scrollZoom?: boolean;
+  /** Whether the map cannot be moved. */
+  cannotMoveMap?: boolean;
```

Comment:
Not sure about the naming... What about `disableInteractions`?

Reply:
Yes, seems better :)